pkg/apiutil/apiutil.go

```diff
@@ -68,3 +65,22 @@ func ReadJSON(r io.ReadCloser, data interface{}) error {
 
 	return err
 }
+
+// FieldError connects an error to a particular field
+type FieldError struct {
+	error
+	field string
+}
+
+// ParseVarUint wraps strconv.ParseUint with detailed error reporting
+func ParseVarUint(vars map[string]string, varName string, base int, bitsize int) (uint64, error) {
```

Comment:
em, VarUint here may confuse users. I think it is to parse Varint at first...

Reply:
Sorry, I don't understand this comment. I could rename this to `VarsParseUint` to better mirror the underlying function name.